torchvision/transforms/v2/functional/_color.py

```diff
@@ -641,3 +643,43 @@ def invert_image_tensor(image: torch.Tensor) -> torch.Tensor:
 @_register_kernel_internal(invert, datapoints.Video)
 def invert_video(video: torch.Tensor) -> torch.Tensor:
     return invert_image_tensor(video)
+
+
+@_register_explicit_noop(datapoints.BoundingBoxes, datapoints.Mask)
+def permute_channels(inpt: datapoints._InputTypeJIT, permutation: List[int]) -> datapoints._InputTypeJIT:
```

Comment:
Needs docs, in particular for the `permutation` param

Reply:
I've added a docstring, but so far the v2 functionals are not part of our HTML documentation. LMK how you want me to include this.

Reply:
Thanks for the docstring and sorry I wasn't clear enough. The key thing to document (and that is still missing) is what the `permutation` parameter really is. In particular it's not clear whether 

```permutation=[2, ...]```

means "put index 0 at position 2" or "put index 2 at position 0". Right now, only the tests allow to understand that.